### PR TITLE
#50 Revision of 00 Intro / 01 Import / README.md

### PR DESCRIPTION
**Pull request by fjcalzado**
- Typos fixing.
- Indentation review.
- Code block review.
- Reformatted and improved Appendix content.
- Steps followed. It woks as expected.

### CODE REVIEW

File: 00 Intro/01 Import/README.md [hunk 1]
```diff
@@ -1,149 +1,174 @@
...
-npm install
-```
+  ```
+  npm install
+  ```
 
-- Let's add a new file called averageService.js, this file will contain a function that will calculate the average value of a given array, this function will be exported (make it visible to other modules that need to consume them).
+- Let's add a new file called `averageService.js`. This file will contain a function that will calculate the average value of a given array, this function will be exported (make it visible to other modules that need to consume them).
 
 ### ./averageService.js
```
Comment (nasdan) [inline 00 Intro/01 Import/README.md]: Please, move this line next to code block to improve readability

File: 00 Intro/01 Import/README.md [hunk 2]
```diff
@@ -1,149 +1,174 @@
...
+  - Multiple exports usage in `averageService.js`:
 
-Import * and usage sum + average
+    ```diff
+    - export default function getAvg(scores) {
+    + export function getAvg(scores) {
+    return getTotalScore(scores) / scores.length;
+    }
 
-```diff
```
Comment (nasdan) [inline 00 Intro/01 Import/README.md:132]: It's important to write file path before code block to help reader

File: 00 Intro/01 Import/README.md (hunk 1 above)
Comment (fjcalzado) [reply 00 Intro/01 Import/README.md]: Hi Dani, sorry but I don't get it. Do you mean moving the title "./averageService.js" closer to the code block below? They are already next to each other.

File: 00 Intro/01 Import/README.md (hunk 2 above)
Comment (fjcalzado) [reply 00 Intro/01 Import/README.md:132]: I don't get this point either Dani :(. The code block you refer to is already preceded by the caption:

Multiple exports usage in averageService.js:
Do you mean to add the path of the file (e.g.: ./averageService.js) on top of each code block?

Comment (nasdan) [reply 00 Intro/01 Import/README.md:132]: Yes, exactly, for example here:

### ./averageService.js
```diff

export default ....
....
```

File: 00 Intro/01 Import/README.md (hunk 1 above)
Comment (nasdan) [reply 00 Intro/01 Import/README.md]: Hi @fjcalzado, I mean for example:

- Add the following content to `averageService.js`:

### ./averageService.js
```javascript
export function ....
....
```

**Comment by fjcalzado** [comment]
Ok, added file path on top of every code block in bc2e00d1127af6b047ab40832cbaf71950fd310b

**Comment by nasdan** [comment]
Thanks a lot @fjcalzado.
Last comment: do you think we need to update this [Readme](https://github.com/Lemoncode/webpack-2.x-by-sample/blob/master/README.md)? I mean, root Readme has same summary info like _00 Intro/01 Import/Readme.md_

**Comment by fjcalzado** [comment]
Sure @Nasdan , done it in cc8a1c388c0605e8a3911fcbbd5bfa86893252b4.
I have just copied the summary from 00 Intro / 01 Import readme to the root readme. However, you will notice that I have wrapped every reference to a file with backticks, so they get highlighted in presentation. The thing is, in order to be consistent, we should either keep this notation in the rest of the root readme summaries or remove it from my summary part. Whatever you prefer, I can take care of that.

**Comment by nasdan** [comment]
@fjcalzado great. I like your contribution. What do you think if we create a new issue to change other samples like this notation and assign it to you?

**Comment by fjcalzado** [comment]
Sure! No problem, go ahead!

**Comment by nasdan** [comment]
Nice, I go to merge and create new issue

**Pull request merged**